Sources/tart/Commands/Create.swift

```diff
@@ -28,6 +31,16 @@ struct Create: AsyncParsableCommand {
         throw ValidationError("Only Linux VMs are supported on Intel!")
       }
     #endif
+
+    // Validate disk format support
```

Comment:
1. Let's have the same validation in `tart run` since only macOS Tahoe can work with ASIF.

2. Since `diskFormat.isSupported` is the same thing as `diskFormat.canCreate`, we can probably remove the latter.